public/js/common/common.js

```diff
@@ -1,5 +1,13 @@
 var midas = midas || {};
 midas.slicerappstore = midas.slicerappstore || {};
+'use strict';
+
```

Comment:
Add a comment indicating that this is expected to be true if Qt5.x where Web?Engine is used